src/main/kotlin/net/maizegenetics/phgv2/cli/CreateRefVcf.kt

```diff
@@ -190,14 +187,12 @@ class CreateRefVcf : CliktCommand(help = "Create and load to tiledb a haplotype
                     // Only add to the alt header lines if the intervalHash is not already in the set.
                     // There are duplicate sequences in the reference file when splitting by gff defined genes
                     if (!currentSeqHashs.contains(intervalHash)) {
-                        altHeaderLines.add(
-                            VCFAltHeaderLine(
-                                "<ID=${intervalHash}, Description=\"haplotype data for line: ${refName}\"," +
-                                        "Source=\"${refGenome}\",SampleName=\"${refName}\",Regions=\"${chr}:${anchorStart}-${anchorEnd}\"," +
-                                        "Checksum=\"${intervalHash}\",RefChecksum=\"${intervalHash}\",RefRange=\"${chr}:${anchorStart}-${anchorEnd}\">",
-                                VCFHeaderVersion.VCF4_2
-                            )
-                        )
+                        val altHeaderLine = StringBuilder().apply {
```

Comment:
Is there a reason to change this to a StringBuilder?  The template that it was doing before should be fine.

Reply:
Yes, the StringBuilder() is more efficient as it is not creating a new String object with each concatenation.  This code was changed based on some junit code (for creating the LIneB.h.vcf file) that ended up not being needed.

Reply:
The only reason for the concatenation is just for readability in the IDE.  My hunch is the kotlin compiler will merge this together.  I feel that it is much more readable without the StringBuilder as the String with templating is much simpler.